<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Software and serial ports</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>Serial port</string>
          </property>
         </widget>
        </item>
				<item row="0" column="1">
				 <widget class="QComboBox" name="combobox_port"/>
				</item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_serial">
          <property name="text">
           <string>Show serial monitor</string>
          </property>
         </widget>
        </item>
				<item row="0" column="4">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
				<item row="0" column="5">
				 <widget class="QLabel" name="label_6">
					<property name="text">
					 <string>Software version</string>
					</property>
				 </widget>
				</item>
				<item row="0" column="6">
				 <widget class="QLineEdit" name="lineEdit_software"/>
				</item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
	 
	 <item>
    <widget class="QTabWidget" name="scd_widget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_update">
       <property name="text">
        <string>Update plot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ak1">
       <property name="text">
        <string>           </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_correct">
       <property name="text">
        <string>Correct data</string>
       </property>
       <property name="checkable">
        <bool>true</bool> 
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ak1">
       <property name="text">
        <string>           </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_calibration">
       <property name="text">
        <string>View calibration curve</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ak1">
       <property name="text">
        <string>           </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_voltage_noise">
       <property name="text">
        <string>View voltage curve</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>   
   
   <item column="0">
    <widget class="QGroupBox" name="acknowledgments">
     <property name="title">
      <string>Acknowledgment</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_ak1">
          <property name="text">
           <string>Â©   L.Teruzzi, L.Cremonesi, M.A.C. Potenza --- Department of Physics, University of Milan, Milan (IT)                                                                                                                               </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_ak2">
          <property name="text">
           <string>Thanks to: European Cold Laboratory Facilities, Department of Earth and Environmental Sciences, University of Milano-Bicocca, Milan (IT)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_help">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
